src/main/resources/locales/en/subscription-configure-list-confirmed.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Subscriptions updated",
+  "header": "Subscriptions updated",
+  "subheader": "Your subscription(s) have been updated successfully",
+  "p1": "To continue, you can go to ",
+  "link_text": "your account",
+  "p2": " in order to:",
+  "addEmailSubscription": "add a new email subscription",
+  "manageYourCurrentSubscriptions": "manage your current email subscriptions",
+  "manageYourListTypeSubscriptions": "Further refine the list types per court received",
```

Comment:
This is part of the bullet points like the other lines but this starts with an upper case but the other ones start with lower case. Consider making them consistent?

Reply:
As per discussion with Chris C, we do not need to include third bullet point. So, I have removed and and updated the rest of content changes on the page.